cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,137 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+  cy.visit('/');
+});
+
+it('Pagination on the page', () => {
+  cy.contains('.-pagination', 'Previous')
+    .should('exist');
+  cy.contains('.-pagination', 'Next')
+    .should('exist');
+  cy.contains('.-pagination', 'Page')
+    .should('exist');
+  });
+
+it('Should have the ability to add worker', () => {
+  cy.contains('.btn', 'Add')
+    .click();
+  cy.get('#firstName')
+    .type('Rosa');
+  cy.get('#lastName')
+    .type('Winters');
+  cy.get('#userEmail')
+    .type('rowisa@yep.com');
+  cy.get('#age')
+    .type('18');
+  cy.get('#salary')
+    .type('5600');
+  cy.get('#department')
+    .type('Cepka');
+  cy.get('#submit')
+    .click();
+
+  cy.contains('.rt-td', 'Rosa', 'Winters', '18', '5600', 'Cepka')
```

Comment:
The function `.contains()` requires two arguments, string and text to match. Here you are trying to pass multiple string values which may cause unintended behavior. Consider using multiple `.contains()` assertions for different values.